src/Avalonia.Base/Layout/ContainerType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Avalonia.Layout
+{
+    /// <summary>
+    /// Defines how a container is queried.
+    /// </summary>
+    public enum ContainerType
+    {
+        /// <summary>
+        /// The container will not be queries for any container size queries.
```

Comment:
fix: The container will not be querie`d` for any container size queries.
better: The container is not included in any size queries.